src/ring_theory/power_basis.lean

```diff
@@ -276,6 +291,19 @@ pb.constr_pow_gen hy
   pb.lift y hy (aeval pb.gen f) = aeval y f :=
 pb.constr_pow_aeval hy f
 
+/-- `pb.lift_equiv` states that roots of the minimal polynomial of `pb.gen` correspond to
+maps sending `pb.gen` to that root.
+
+This is the bundled equiv version of `power_basis.lift`.
+-/
+@[simps]
+noncomputable def lift_equiv [nontrivial S] (pb : power_basis A S) :
```

Comment:
This and/or `alg_hom_ext` might be worth mentioning in the docstring

Reply:
Oh, do you need `nontrivial` here?

Reply:
Good point. In fact, if we tweak one proof slightly, we can delete almost all `nontrivial` assumptions in this file!